public/summer-slam-2017/index.html

```diff
@@ -0,0 +1,539 @@
+              <br/>
+              <br />
+              We've got some <a href="/code-of-conduct/" title="If you have any trouble during the event, contact one of us...our names are on our code of conduct page">new faces</a> on the Indy Hackers board (including some old faces) and we're trying a lot of new things to help bring the greater Indianapolis Hacker community together. Like the Summer Slam. We hope you'll join us, and have a great time.
+              <br/>
+              <br />
+            </p>
+
+            <div class="layout-50 clear-fix">
+
+              <!-- Left column -->
+              <div class="layout-50-left clear-fix">
+
+                <h2>I'm a "home for hackers"...</h2>
+
+                <!-- Features list -->
+                <ul class="no-list list-2 clear-fix">
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Hiring a Developer?</h3>
+                    <p class="icon-1 icon-1-6">
+                      The single best way to hire a developer, is by being connected to the community. If we know you, and how awesome your mission is, how incredible you work environment is, and how much love you have for the community at large, I guarantee you we will think of you first when we are looking for our next adventure.
+                    </p>
+                  </li>
+                  <!-- /Item -->
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Increase Awareness of your Company?</h3>
+                    <p class="icon-1 icon-1-6">
+                      Sometimes it isn't about recruiting...sometimes you are just proud of what you are doing, and want others in the tech world to be more aware of it. This is a great opportunity to show us what you have cooking!
+                    </p>
+                  </li>
+                  <!-- /Item -->
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Just want to support Local Developer Community?</h3>
+                    <p class="icon-1 icon-1-3">
+                      We all know what the world thinks of "West Coast Engineers" and "East Coast Coders"...well, we are incredibly proud of what we are doing right here in the mighty MidWest. If you want to help us showcase that, supporting events like this goes a long way towards that goal.
+                    </p>
+
+                  </li>
+                  <!-- /Item -->
+
+                </ul>
+                <!-- /Features list -->
+
+
+                <p class="subtitle-paragraph">
+                  For more information on how to become a sponsor, check out <a target="__BLANK" href="/summer-slam-2017/image/indyhackers.org_holiday_social_2016_sponsorships.pdf">the Sponsorship PDF</a> and/or just <a target="__BLANK" href="mailto:sponsors@indyhackers.org?subject=Sponsoring-Indy-Hackers-Holiday-Social">e-mail us</a>! Absolutely all sponsor money goes toward the event.
```

Comment:
I haven't made this thing yet. Should probably start a discussion about it. I think we could remove the sponsor sheet thing to just "email us" for now.